### ISSUE TITLE
Meta-D - Delete the content under the cursor until the next word boundary to the right

### ISSUE THREAD

**Issue by caribpa** [rizinorg/rizin#767]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits (mandatory)                             | Arch Linux x64
| File format of the file you reverse (mandatory)      | n/a
| Architecture/bits of the file (mandatory)            | n/a
| `rizin -v` full output, **not truncated** (mandatory)  | rizin 0.2.0-git @ linux-x86-64, package: 0.2.0.r25903.882795689-git (AUR) commit: 88279568978233385838e1b0e6f320aa0187de9a, build: 2021-03-04__19:45:29

### Expected behavior
Pressing `Meta-D` deletes the content under the cursor until the next word boundary to the right (opposite of `Ctrl-W`). This is the standard `Meta-D` behavior in all mainstream shells, readline, and emacs.

### Actual behavior
Pressing `Meta-D` performs unexpected behavior (sometimes it deletes to the right, but not the amount expected, other times it transposes text or pastes something unexpected).

### Steps to reproduce the behavior
```sh
# [ ] denotes the current cursor position
$ rizin --
> this is a test[ ]

# *Presses Meta-B twice*
> this is [a] test

# *Presses Meta-D*
> this is [ ]ttest  # wut?

# *Furiously presses Meta-D again*
> this is [ ]       # hmmm
```

### Additional Logs, screenshots, source code,  configuration dump, ...
This behavior is also present in radare2.

**Comment by ret2libc** [comment]
Thanks for creating this. It was actually in my TODO for a very long time because to me it is super annoying!! I will fix this for 0.3.0, when, I think, I will focus on general shell/usage improvements.

**Comment by caribpa** [comment]
@ret2libc, I feel your pain (*hugs @ret2libc and both cry* 😭)
